test/Controller/ArticleControllerTest.php

```diff
@@ -1471,45 +1453,26 @@ public function it_displays_rds_info_bar_when_it_has_associated_rds()
         );
         $this->assertNotEmpty($crawler->filter('.article-download-links-list__link')->selectLink('Executable version'));
         $this->assertNotEmpty($crawler->filter('.article-download-links-list__secondary_link')->selectLink('What are executable versions?'));
-        $this->assertContains('/id-of-article-with-rds/executable/download', $crawler->filter('.article-download-links-list__link')->selectLink('Executable version')->attr('href'));
+        $this->assertContains('/id-of-article-with-era/executable/download', $crawler->filter('.article-download-links-list__link')->selectLink('Executable version')->attr('href'));
         $this->assertContains('Executable code', $crawler->filter('.view-selector')->text());
     }
 
     /**
      * @test
      * @backupGlobals enabled
      */
-    public function it_does_not_displays_rds_info_bar_when_it_has_associated_rds_but_it_is_not_the_latest_version()
+    public function it_does_not_displays_era_info_bar_when_it_has_associated_era_but_it_is_not_the_latest_version()
```

Comment:
typo in function name should be `it_does_not_display_era_info_bar_when_it_has_associated_era_but_it_is_not_the_latest_version`